docs/docs/spec.md

```diff
@@ -81,6 +24,52 @@ This document also links to other documents. Those documents are non-normative,
 except for when we link to the Java Language Specification to defer to its
 rules.
 
+> ### The word "nullable"
+>
+> In this doc, we aim not to refer to whether a type "is nullable." Instead, we
+> draw some distinctions, creating at least four kinds of "Is it nullable?"
+> questions we can ask for any given type usage:
```

Comment:
```suggestion
> define at least four kinds of "Is it nullable?"
> questions we can ask for any given type usage:
```

Also, why hedge with "at least"? Are there more than four?

Reply:
I'll update it to say "four."